phpunit.xml.dist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<phpunit
```

Comment:
Can you use phpunit configuration matching DBAL? https://github.com/doctrine/dbal/blob/master/phpunit.xml.dist#L14
Also no verbose by default please.